match/lib/match/utils.rb

```diff
@@ -9,9 +9,18 @@ def self.import(item_path, keychain)
       # as the keychain path.
       #
       # We need to expand each path because File.exist? won't handle directories including ~ properly
+      #
+      # We also try to append `-db` at the end of the file path, as with Sierra the default Keychain name
+      # has changed for some users:
+      # - https://github.com/fastlane/fastlane/issues/5649
```

Comment:
Do we need to link to all of these issues in the comments? I feel like adding all of these here clutters up the code quite a bit.